[easy] Remove duplicate source of truth for database migrations

ttung opened the pull request:
### Description
The revision id and downward revision is in the python code a few lines down.

### Test plan
These are comments, YOLO.

phoenixAja reviewed (commented):
interesting, I'm still a little confused why you want to remove this though

phoenixAja commented:
just read your description again, so ignore the last comment.

Commit:
Remove duplicate source of truth for database migrations

### Description
The revision id and downward revision is in the python code a few lines down.

### Test plan
These are comments, YOLO.
